Graphics/GraphicsEngine/interface/DeviceContext.h

```diff
@@ -1071,6 +1071,14 @@ DILIGENT_BEGIN_INTERFACE(IDeviceContext, IObject)
     
     /// \param [in] Attribs        - Structure describing the command attributes, see Diligent::DrawMeshIndirectAttribs for details.
     /// \param [in] pAttribsBuffer - Pointer to the buffer, from which indirect draw attributes will be read.
+    ///                              The buffer must contain the following arguments at the specified offset:
```

Comment:
Not sure I completely understand the format that would work for both:
D3D12 needs (TaskCount, 1, 1), while Vulkan expects (TaskCount, 0)

Reply:
> D3D12 needs (TaskCount, 1, 1), while Vulkan expects (TaskCount, 0)
yes